src/user-guide/writing-suites.rst

```diff
@@ -182,6 +182,8 @@ configuration items grouped under several top level section headings:
 
 :cylc:conf:`[cylc]`
    Non task-specific suite configuration.
+:cylc:conf:`[scheduler]`
```

Comment:
I have kept the style of the pre-existing sections, although, would it be better for these to be replaced with exact links to the spec like... `:cylc:conf:`flow.cylc[scheduler]`` - although perhaps this does not look as aesthetically pleasing?